Snowflake.Data/Core/Session/SFSession.cs

```diff
@@ -145,6 +159,8 @@ internal SFSession(String connectionString, SecureString password)
                 restRequester = new RestRequester(_HttpClient);
                 extractedProperties.WarnOnTimeout();
                 connectionTimeout = extractedProperties.TimeoutDuration();
+                properties.TryGetValue(SFSessionProperty.CLIENT_CONFIG_FILE, out var easyLoggingConfigFile);
+                _easyLoggingStarter.Init(easyLoggingConfigFile);

```

Comment:
`easyLoggingConfigFile` is created using `TryGetValue`, so I believe it can be null. I assume it safe to call it like here?

Reply:
Sure, it's safe. Specifying CLIENT_CONFIG_FILE is just one of possibilities. Other possible usecases are that the file can be taken from environment variable or predefined directory. So it's completely fine if it's null.